docs/01-jwt.md

```diff
@@ -743,7 +743,7 @@ func ExampleJWT_ValidateIssuer() {
   }
   fmt.Printf("%s\n", err)
   // OUTPUT:
-  // "iss" not satisfied: values do not match
```

Comment:
This doesn't hurt, but ... why? I don't know, it doesn't strike me as something that's important

Reply:
I changed the message just to be consistent with the other similar errors. However, since the `Validate()` function is expected to return always the same error type with a different message depending on the case (existence of the claim, invalid value, etc), I can definitely revert the change